foundry/preprocessing/sklearn.py

```diff
@@ -156,8 +157,15 @@ def as_transformer(x: TransformerLike) -> TransformerMixin:
     else:
         raise TypeError(f"{type(x).__name__} does not have a `transform()` method.")
 
+class make_column_selector(make_column_selector_sklearn):
+    def __repr__(self):
+        return f"make_column_selector(pattern={self.pattern}, dtype_include={self.dtype_include}, dtype_exclude={self.dtype_exclude})"
 
-def make_column_dropper(
+def make_column_dropper(*args, **kwargs):
+    warn("make_column_dropper is deprecated, use make_drop_transformer")
+    return make_drop_transformer(*args, **kwargs)
```

Comment:
I give my official authorization to omit this, as I know everyone using this package and I know no one except you is using `make_column_dropper`